src/main/java/at/hannibal2/skyhanni/data/ProfileStorageData.kt

```diff
@@ -67,34 +33,28 @@ object ProfileStorageData {
             return
         }
         if (sackPlayers == null) {
-            LorenzUtils.error("sackPlayers is null in sackPlayers!")
+            LorenzUtils.error("sackPlayers is null in ProfileJoinEvent!")
             return
         }
 
-        if (profileSpecific == null) {
-            val profileName = event.name
-            loadProfileSpecific(playerSpecific, sackPlayers, profileName)
-        }
+        val profileName = event.name
+        loadProfileSpecific(playerSpecific, sackPlayers, profileName)
+        ConfigLoadEvent().postAndCatch()
     }
 
     @SubscribeEvent
     fun onTabListUpdate(event: TabListUpdateEvent) {
-        if (profileSpecific != null) return
-        val playerSpecific = playerSpecific ?: return
-        val sackPlayers = sackPlayers ?: return
+        if (!LorenzUtils.inSkyBlock) return
+
         for (line in event.tabList) {
-            val pattern = "§e§lProfile: §r§a(?<name>.*)".toPattern()
+            val pattern = "§[ed]§lProfile: §r§a.*".toPattern()
```

Comment:
dont like 
a) that this pattern is being made for each line in the tablist
b) that it is the exact same pattern as in the HypixelData file, maybe move the pattern to UtilsPattern and reference both from there

Reply:
Yeah, I tried to reuse the pattern from the other file but it caused a weird test failure with visitors. I will try with UtilsPattern.

Reply:
> Yeah, I tried to reuse the RepoPattern from the other file but it caused a weird completely unrelated test failure with visitors. I will try with UtilsPattern.

I'm 99% sure i fixed test failures since then